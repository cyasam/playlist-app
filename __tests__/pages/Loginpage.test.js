import React from 'react'
import { shallow } from 'enzyme'
import { Loginpage } from '../../src/scripts/pages/Loginpage'

describe('Loginpage Component', () => {
  const setup = () => {
    const wrapper = shallow(<Loginpage />)

    return {
      wrapper
    }
  }

  it('renders properly', () => {
    const wrapper = setup()
    expect(wrapper).toMatchSnapshot()
  })
})
