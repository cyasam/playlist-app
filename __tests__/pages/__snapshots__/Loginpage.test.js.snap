// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loginpage Component renders properly 1`] = `
Object {
  "wrapper": <SideEffect(DocumentTitle)
    title="Login - Playlist App"
  >
    <withRouter(Connect(LoginForm)) />
  </SideEffect(DocumentTitle)>,
}
`;
